### PR TITLE
rename Jetpack.io to Jetify.io

### PR DESCRIPTION
**Pull request by alexravenna**
## What type of PR is this?

- [ ] Add a new dev container collection
- [ ] Update to an existing dev container collection
- [x] Documentation/spec update
- [ ] Other containers.dev site update (UX, layout, etc)

### CODE REVIEW

File: supporting.md [hunk 1]
```diff
@@ -65,9 +65,9 @@ Cachix's **[devenv](https://devenv.sh/)** now supports automatically generating
 
 See [devenv documentation](https://devenv.sh/integrations/codespaces-devcontainer/) for detais. 
 
-### <a href="#jetpack-io-devbox" name="jetpack-io-devbox" class="anchor"> Jetpack.io Devbox </a>
+### <a href="#jetify-io-devbox" name="jetify-io-devbox" class="anchor"> Jetify.io Devbox </a>
 
-[Jetpack.io](https://jetpack.io) is a [Nix](https://nixos.org/)-based service for deploying applications. [DevBox](https://www.jetpack.io/devbox/) provides a way to use Nix to generate a development environment. [Jetpack.io's VS Code extension](https://marketplace.visualstudio.com/items?itemName=jetpack-io.devbox) allows you to quickly take advantage of DevBox in any Dev Container Spec supporting tool or service. 
+[Jetify.io](https://jetify.io) (formerly jetpack.io) is a [Nix](https://nixos.org/)-based service for deploying applications. [DevBox](https://www.jetify.io/devbox/) provides a way to use Nix to generate a development environment. [Jetify.io's VS Code extension](https://marketplace.visualstudio.com/items?itemName=jetpack-io.devbox) allows you to quickly take advantage of DevBox in any Dev Container Spec supporting tool or service.
```
Comment (samruddhikhandale) [inline supporting.md]:
```suggestion
[Jetify.io](https://jetify.io) (formerly jetpack.io) is a [Nix](https://nixos.org/)-based service for deploying applications. [DevBox](https://www.jetify.com/devbox/) provides a way to use Nix to generate a development environment. [Jetify.io's VS Code extension](https://marketplace.visualstudio.com/items?itemName=jetpack-io.devbox) allows you to quickly take advantage of DevBox in any Dev Container Spec supporting tool or service. 

```

https://www.jetify.io/devbox/ doesn't open for me 🤔

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks for the spec update ⚡ 

Left one comment, thanks!

File: supporting.md (hunk 1 above)
Comment (alexravenna) [reply supporting.md]: Thanks for the catch! 😊 My bad...they're called just "Jetify", not "Jetify.io". I corrected the text accordingly 🙂.

**Review by samruddhikhandale — APPROVED** [review]
Thanks! ✨

**Pull request merged**